examples/bridge-app/linux/include/Clusters.h

```diff
@@ -0,0 +1,294 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/AttributeAccessInterface.h>
+#include <app/ConcreteAttributePath.h>
+#include <app/reporting/reporting.h>
+#include <app/util/attribute-storage.h>
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <functional>
+
+static constexpr uint16_t ZCL_DESCRIPTOR_CLUSTER_REVISION              = 1;
```

Comment:
Why are these here?

Reply:
Default initialization of the cluster revision attribute

Reply:
This file is too much a mash-up of library APIs, and cluster-specific bits. Please separate.